src/utils/useSignIn.ts

```diff
@@ -0,0 +1,141 @@
+import { useState } from "react";
+
+// @ts-expect-error File is in plain JS
+import { sha256 } from "./sha256";
+
+const CHROMATIC_BASE_URL = "https://www.staging-chromatic.com";
```

Comment:
This will need to be configurable or to be updated to https://www.chromatic.com
I set it to the review app for testing purposes. Not sure how other addons deal with this sort of thing, but figma-addon has uses an environment variable:

`const apiUrl = process.env.STORYBOOK_CHROMATIC_API_URL || "https://www.chromatic.com";`

Reply:
Yep this was on my TODO list. At some point I got a "process is not defined" error but maybe that was a red herring.